python/mxnet/ndarray.py

```diff
@@ -1104,48 +1103,41 @@ def arange(start, stop=None, step=1.0, repeat=1, ctx=None, dtype=mx_real_t):
     """Returns evenly spaced values within a given interval.
 
     Values are generated within the half-open interval [start, stop). In other
-    words, the interval includes start but excludes stop. For integer
-    arguments, the function is equivalent to the built-in Python function ``range``
-    and to ``numpy.arange``, but returns an ``NDArray``.
+    words, the interval includes start but excludes stop. The function is
```

Comment:
`start` and `stop` are parameters names so they should appear in backticks... (also a change suggested in earlier code review)